File: backend/python_main/db/db_config_sample.py
# db_config.py

# 데이터베이스 연결 정보
DB_CONFIG = {
    'host': 'localhost',
    'user': 'username',
    'password': 'password',
    'database': 'databasename'
}

# 데이터베이스 연결 정보를 반환하는 함수
def get_db_config():
    return DB_CONFIG

File: backend/disaster_msg/db_config_sample.php
<?php
// db_config.php
$host = 'localhost';  // DB 호스트
$db   = 'DRPS';  // DB 이름
$user = 'username';  // DB 사용자명
$pass = 'password';  // DB 비밀번호

// PDO 사용하여 DB 연결
try {
    $pdo = new PDO("mysql:host=$host;dbname=$db;charset=utf8", $user, $pass);
    $pdo->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
} catch (PDOException $e) {
    die("DB 연결 실패: " . $e->getMessage());
}
?>

